content/client-server-api/modules/threading.md

```diff
@@ -12,11 +12,11 @@ as by providing some context to what is going on in the thread but keeping the f
 history behind a disclosure.
 
 Threads are established using a `rel_type` of `m.thread` and reference the
-*thread root* (the first event in a thread). It is not possible to create a
-thread from an event which itself is the child of an event relationship (i.e.,
-one with an `m.relates_to` property). It is therefore also not possible to nest
-threads. All events in a thread reference the thread root instead of the
-most recent message, unlike rich reply chains.
+*thread root*. It is not possible to create a thread from an event which itself
```

Comment:
Attempted a definition in 058e97e5 . I removed the original definition because it was not accurate (because it said the thread root was in the thread, which it is not).